### PR TITLE
LIB-145 Updated Dispensing for with OPD, locations, and license #

### PR DESCRIPTION
**Pull request by lnball**
@mogoodrich I'd really like Tommy to review, but he's not listed as a reviewer.  How do I add him to this repo?

### CODE REVIEW

**Comment by lnball** [comment]
Look over the code @tommyiversonj .  Let me know if you want to do it together.

**Comment by mogoodrich** [comment]
Looks like you got @tommyiversonj added!

**Comment by lnball** [comment]
@mogoodrich I did add @tommyiversonj .  It's been awhile since I've done this and had permissions.  Hopefully he can review the code here.  I'm around tomorrow to go over it...

File: configuration/pih/htmlforms/dispensing.xml [hunk 1]
```diff
@@ -176,33 +208,50 @@
...
                 </label>
+                <obs conceptId="PIH:6592"/>
+
+                <p class="side-by-side">
+                    <label>
+                        <uimessage code="Title" />
+                    </label>
+                    <obs conceptId="CIEL:163556" style="radio"
+                         answerConceptIds="CIEL:162591,CIEL:1577,CIEL:162592"
+                         answerCodes="Dr,RN,PA" />
```
Comment (tommyiversonj) [inline configuration/pih/htmlforms/dispensing.xml:245]: Can Nurse Aid be added? Nurse Aids are unlicensed assistive personnel.

**Pull request merged**